.vscode/launch.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "0.2.0",
+    "version": "0.3.0",
```

Comment:
Nit: I don't think this was referring to the tracker version :D

Reply:
🙈